### PR TITLE
CHECKOUT-929 WIP: image upload functionality

### PR DESCRIPTION
**Pull request by jordanwink201**
allow for custom header bg and custom logo for uco

ng-stencil-editor: https://github.com/bigcommerce-labs/ng-stencil-editor/pull/133

@bigcommerce-labs/checkout

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -37,6 +37,10 @@
     "last 3 versions"
   ],
   "settings": {
+    "use_custom_background": false,
```
Comment (PascalZajac) [inline config.json]: Should these settings be scoped to the checkout somehow, maybe with a prefix? I take it nested objects aren't allowed?

Comment (jordanwink201) [inline config.json]: yeah I should add a prefix 'uco_background_image'

**Comment by davidchin** [comment]
LGTM. Just want to double check, do you want to PR against master or a feature branch? (your work for the editor is against a feature branch)

**Comment by jordanwink201** [comment]
@davidchin yeah I could open a PR against jordanwink201:PROJECT-1134 since I can't create a stencil branch like bigcommerce:PROJECT-1134

**Comment by jordanwink201** [comment]
re-open against stencil/PROJECT-1134

**Pull request closed without merging**